lib/stemcell/templates/bootstrap.sh.erb

```diff
@@ -48,6 +49,7 @@ domain_name='<%= opts['instance_domain_name'] %>'
 chef_version='<%= opts['chef_version'] %>'
 username='<%= opts.fetch('user', ENV['USER']) %>'
 encrypted_data_bag_secret_path='<%= opts['chef_data_bag_secret_path'].to_s %>'
+node_json='<%= opts['node_json'] %>'
```

Comment:
the quoting would break if the node json contains `'` characters. this requires a more serious escaping strategy -- a HEREDOC might to the trick.

Reply:
Good point, I've added a commit that solves this.